rllib/tuned_examples/a2c/cartpole_a2c.py

```diff
@@ -1,3 +1,6 @@
+# Run with:
+# rllib train -f cartpole_a2c.py --type python \
```

Comment:
Sorry, dumb question: I thought the -f option was no longer supported, instead we have to do:
```
$ rllib train file [my file].py --type python
```

now.

?

Reply:
![Uploading image.png…]()